base/initdefs.jl

```diff
@@ -18,6 +18,12 @@ An array of the command line arguments passed to Julia, as strings.
 """
 const ARGS = String[]
 
+"""
+    exit([code])
+
+Quit (or control-D at the prompt). The default exit code is zero, indicating that the
+processes completed successfully.
```

Comment:
Sounded a little off on first read, yes. Technically more correct than a singular equivalent though I suppose.

On that note, the "control-D at the prompt" snippet strikes me as out of place as well. Perhaps could be listed as a synonym instead? Also could use a ref to the `quit()` docstring.